oat-sa/extension-tao-testqti
# Feature/tao 10377/cloud front modify links to assets qti test package before import

__Related to task:__ https://oat-sa.atlassian.net/browse/TAO-10377
__Description:__ If CloudFront is configured run TAO script to modify links to assets QTI test package before import

It depends on it https://github.com/oat-sa/extension-tao-operations/pull/122

__how to test:__
1. create s3 backet and cloudFront on aws and and configure it as per doc https://github.com/oat-sa/extension-tao-operations/wiki/Create-CloudFront-Package
2. configure files replacement.conf.php and awsClient.conf in config/generis according to your aws account
3. make test import
As a result, a folder with static items data should be created in your s3 backet, static data of created items must reference your s3 backet
